README.md

```diff
@@ -96,6 +107,19 @@ while read -r vm; do
 done <<< "$VMS"
 ```
 
+#### Using Azure AD Application ID and Secret
```

Comment:
Could you add a note to modify the Deployment to add the secrets as Environment variables and add a small example of what it would look like?  Having a sample of deploying the secret would be helpful as well.